loader/dd_library_loader.c

```diff
@@ -712,7 +746,7 @@ static int ddloader_api_no_check(int api_no) {
         default:
             if (!force_load || api_no < 320151012) {
                 telemetry_reason reason = api_no < 320151012 ? REASON_EOL_RUNTIME : REASON_INCOMPATIBLE_RUNTIME;
-                TELEMETRY(reason, NULL, "Found incompatible runtime (api no: %d). Supported runtimes: PHP 7.0 to 8.3", api_no);
+                TELEMETRY(reason, NULL, NULL, "Found incompatible runtime (api no: %d). Supported runtimes: PHP 7.0 to 8.4", api_no);
```

Comment:
Small note: Maybe we should put
```
#define MIN_API_VERSION 320151012
#define MAX_API_VERSION 420240924
#define MIN_PHP_VERSION 7.0
#define MAX_PHP_VERSION 8.4
```
here instead of the switch (there realistically anyway no "intermediary" api versions exist)
Making it easier to not forget about updating values in future.